### PR TITLE
Droth 5 default width to new links from vvh

### PR DESCRIPTION
**Pull request by oskarmalmstrom**

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MaintenanceService.scala [hunk 1]
```diff
@@ -191,19 +179,12 @@ class MaintenanceService(roadLinkServiceImpl: RoadLinkService, eventBusImpl: Dig
 
     //Remove the asset ids ajusted in the "linearAssets:update" otherwise if the "linearAssets:saveProjectedLinearAssets" is executed after the "linearAssets:update"
     //it will update the mValues to the previous ones
+    //já temos o roadLink, poderá ser enviado também?
```
Comment (joanacoelho) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MaintenanceService.scala]: Portuguese comment, please remove

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/RoadWidthServiceSpec.scala [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+
+  val mockRoadLinkService = MockitoSugar.mock[RoadLinkService]
+  val mockVVHClient = MockitoSugar.mock[VVHClient]
+  val mockVVHRoadLinkClient = MockitoSugar.mock[VVHRoadLinkClient]
+  val mockPolygonTools = MockitoSugar.mock[PolygonTools]
+  val mockLinearAssetDao = MockitoSugar.mock[OracleLinearAssetDao]
+  val mockEventBus = MockitoSugar.mock[DigiroadEventBus]
+  val linearAssetDao = new OracleLinearAssetDao(mockVVHClient, mockRoadLinkService)
+
+//  object PassThroughService extends LinearAssetOperations {
```
Comment (joanacoelho) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/RoadWidthServiceSpec.scala]: this should be removed

**Comment by joanacoelho** [comment]
ok

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/PavingServiceSpec.scala [hunk 3]
```diff
@@ -0,0 +1,631 @@
...
+    val municipalityCode = 444
+    val administrativeClass = State
+    val trafficDirection = TrafficDirection.BothDirections
+    val functionalClass = 1
+    val linkType = Freeway
+    val boundingBox = BoundingRectangle(Point(123, 345), Point(567, 678))
+    val assetTypeId = 110
+    val geom = List(Point(428906.372,6693954.166),Point(428867.234,6693997.01),Point(428857.131,6694009.293))
+    val len = GeometryUtils.geometryLength(geom)
+    //      [{"modifiedAt":"19.05.2016 14:16:23","linkId":3056622,"roadNameFi":"Lohjanharjuntie","roadPartNumber":1,"administrativeClass":"State","municipalityCode":444,"roadNumber":1125,"points":[{"x":346005.726,"y":6688024.548,"z":0.0},{"x":346020.228,"y":6688034.371,"z":0.0},{"x":346046.054,"y":6688061.11,"z":0.0},{"x":346067.323,"y":6688080.86,"z":0.0}],"verticalLevel":0,"maxAddressNumberRight":1365,"trafficDirection":"TowardsDigitizing","minAddressNumberRight":1361,"roadNameSe":"Lojoåsvägen","functionalClass":4,"linkType":2,"mmlId":1204467577,"modifiedBy":"k638654"}],
```
Comment (marquesfm) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/PavingServiceSpec.scala]: Commented code, delete if not important.

File: digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala [hunk 4]
```diff
@@ -1169,7 +1172,9 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
 
   private def verifyServiceToUse(typeId: Int): LinearAssetOperations = {
```
Comment (leirasf) [inline digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala]: Change this name to `getLinearAssetService(typeId: Int)`.

File: digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala [hunk 5]
```diff
@@ -215,7 +215,16 @@ class IntegrationApi(val massTransitStopService: MassTransitStopService) extends
 
   def linearAssetsToApi(typeId: Int, municipalityNumber: Int): Seq[Map[String, Any]] = {
     def isUnknown(asset:PieceWiseLinearAsset) = asset.id == 0
-    val linearAssets: Seq[PieceWiseLinearAsset] = linearAssetService.getByMunicipality(typeId, municipalityNumber).filterNot(isUnknown)
+    def verifyServiceToUse(typeId: Int): LinearAssetOperations = {
```
Comment (leirasf) [inline digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala]: Change this name to `getLinearAssetService(typeId: Int)`

File: digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/RoadLink.scala [hunk 6]
```diff
@@ -64,14 +71,37 @@ sealed trait SurfaceType {
 }
 
 object SurfaceType {
-  val values = Set(Unknown, None, Paved)
+  val values = Set(Unknown, None, Paved, RoadWidth)
```
Comment (MAC-CGI) [inline digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/RoadLink.scala]: This value "RoadWidth" should be deleted

File: digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/RoadLink.scala [hunk 7]
```diff
@@ -64,14 +71,37 @@ sealed trait SurfaceType {
...
 
   def apply(intValue: Int): SurfaceType = {
     values.find(_.value == intValue).getOrElse(None)
   }
 
   case object Unknown extends SurfaceType { def value = 0 }
   case object None extends SurfaceType { def value = 1 }
-  case object Paved extends SurfaceType { def value = 2 }
+  case object Paved extends SurfaceType { def value = 2}
+  case object RoadWidth extends SurfaceType { def value = 3}
```
Comment (MAC-CGI) [inline digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/RoadLink.scala]: This "RoadWidth" should be deleted, isn't SurfaceType

**Pull request merged**